docusaurus.config.js

```diff
@@ -49,6 +49,20 @@ const config = {
           // Remove this to remove the "edit this page" links.
           editUrl: 'https://github.com/statelyai/docs/tree/main/',
 
+          lastVersion: 'current',
+          includeCurrentVersion: true,
+          versions: {
+            current: {
+              label: 'XState v4',
+              path: 'xstate-v-4',
```

Comment:
Try removing this `path` entirely. Then I think it will default to `/` for the current version 🤞🏻

Reply:
You’re a genius! That worked!

Reply:
(Thank you!)